### PR TITLE
Bug 1121884 - toggle attention toaster on secure app toggled +autoland

### PR DESCRIPTION
**Pull request by alivedise**

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[alivedise Alive.Kuo (alivedise)](https://github.com/alivedise) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=71cb9517b3c5)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[alivedise Alive.Kuo (alivedise)](https://github.com/alivedise) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=626b07ae1c7b)

**Comment by try-server-hook** [comment]
[alivedise Alive.Kuo (alivedise)](https://github.com/alivedise) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a36683fdd541)

File: apps/system/test/unit/attention_toaster_test.js [hunk 1]
```diff
@@ -127,16 +140,47 @@ suite('system/AttentionToaster', function() {
...
+        isFullScreen: function() {
+          return true;
+        }
+      };
+      at1._currentToasterState = 'opened';
+      at1._enter_opened();
+      this.sinon.clock.tick(at1.TOASTER_TIMEOUT);
+      assert.equal(at1._currentToasterState, 'closing');
+    });
+
```
Comment (etiennesegonzac) [inline apps/system/test/unit/attention_toaster_test.js]: one day we should refactor this test file to make it "more external" :)

Comment (alivedise) [inline apps/system/test/unit/attention_toaster_test.js]: (y)

**Pull request merged**